.github/workflows/build.yaml

```diff
@@ -36,6 +36,7 @@ jobs:
           sed -i "s|branch = clean-wire-server-docs|branch = ${CURRENT_BRANCH}|g" .gitmodules
           
           cat .gitmodules
+          make sanitize-pr
```

Comment:
Something is strange here: In line 25 we `cd wire-docs`.

So, I'm under the impression that this job is about building the docs. Sanitizing the PR would in my head be a dedicated job - if we want to load stuff onto Github instead of Concourse (which I would endorse for simple stuff like this.)